packages/toc-extension/src/index.ts

```diff
@@ -93,11 +93,6 @@ async function activateTOC(
     label: trans.__('Run Cell(s)')
   });
 
-  app.contextMenu.addItem({
-    selector: '.jp-tocItem',
-    command: CommandIDs.runCells
-  });
-
```

Comment:
It looks that this PR accidentally removed `runCells` from context menu of ToC.

Reply:
Cross-ref https://github.com/jupyterlab/jupyterlab/issues/11584

Reply:
Getting fixed in #11617.

Reply:
@krassowski Sorry about that! Thanks for fixing this.